### PR TITLE
JBIDE-28476: Upgrade Jboss Maven plugins to 2.7.5

### PR DESCRIPTION
**Pull request by sbouchet**
deprecate discovery-utils and dependency-plugin
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

File: tycho-plugins/target-platform-utils/src/main/java/org/jboss/tools/tycho/targets/FixVersionsMojo.java [hunk 1]
```diff
@@ -112,13 +112,15 @@ public void execute() throws MojoExecutionException, MojoFailureException {
...
-						((TargetDefinitionFile.Unit)unit).setVersion(version);
-					}
+						//((TargetDefinitionFile.Unit)unit).setVersion(version);
+					//}
 				}
 			}
-			TargetDefinitionFile.write(targetDef, new File(targetFile.getParent(), targetFile.getName() + "_fixedVersion.target"));
+			// since no automatic modifications, there is no need to create a new target platform.
+			//TargetDefinitionFile.write(targetDef, new File(targetFile.getParent(), targetFile.getName() + "_fixedVersion.target"));
 		} catch (FileNotFoundException ex) {
```
Comment (sbouchet) [inline tycho-plugins/target-platform-utils/src/main/java/org/jboss/tools/tycho/targets/FixVersionsMojo.java:124]: @jeffmaury question here : as it's not possible to fix versions, should we deprecate this Mojo entirely or provide the information as a warning in console log ?

Comment (jeffmaury) [reply tycho-plugins/target-platform-utils/src/main/java/org/jboss/tools/tycho/targets/FixVersionsMojo.java:124]: We need to find the alternative way of fixing the version as this is used during our TP update releng, no ?

**Review by jeffmaury — CHANGES_REQUESTED** [review]
I don't see discovery-utils and tycho-dependency-plugin folders removal

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**